src/components/Utility/CardHeader/CardHeader.css

```diff
@@ -0,0 +1,6 @@
+.cardheader-container {
+    height: calc(0.875rem + 20px);
+    border-bottom: 1px solid #F3AD28;
+    width: 100%;
+    padding: 10px 1rem 10px;
```

Comment:
No need to set the second 10px. It should function the same as not setting it.

Reply:
You're right! I'll push a mini change for that!